docs/demo/gaiden-css/components/footer/demo.html

```diff
@@ -12,65 +12,79 @@
+        <span class="footer__title">
+          Redes sociais
+        </span>
+        <ul class="footer__list footer__list--line footer__list--social" role="menu" aria-label="Menu rodapé - mídia social">
+          <li class="footer__item footer__item--larger" role="none">
+            <a role="menuitem" aria-label="Link para o Instagram" href="https://www.instagram.com/GetNinjas/" title="Link para o Instagram do GetNinjas." target="_blank" rel="noopener noreferrer">
+              <img class="footer__image" alt="Instagram - @GetNinjas" alt="GetNinjas" src="/demo/gaiden-css/images/icons/instagram.png">
+            </a>
+          </li>
+          <li class="footer__item footer__item--larger" role="none">
+            <a role="menuitem" aria-label="Link para o Facebook" href="https://www.facebook.com/GetNinjas/" title="Link para o Facebook do GetNinjas." target="_blank" rel="noopener noreferrer">
+              <img class="footer__image" alt="Facebook - @GetNinjas" src="/demo/gaiden-css/images/icons/facebook.png">
+            </a>
+          </li>
+          <li class="footer__item footer__item--larger" role="none">
+            <a role="menuitem" aria-label="Link para o Twitter" href="https://www.twitter.com/GetNinjas/" title="Link para o Twitter do GetNinjas." target="_blank" rel="noopener noreferrer">
+              <img class="footer__image" alt="Twitter - @GetNinjas" src="/demo/gaiden-css/images/icons/twitter.png">
+            </a>
+          </li>
+          <li class="footer__item footer__item--larger" role="none">
+            <a role="menuitem" aria-label="Link para o Youtube" href="https://www.youtube.com/channel/UCWgyIuQWHB3u88SY1hr5zZg/featured" title="Link para o Youtube do GetNinjas." target="_blank" rel="noopener noreferrer">
+              <img class="footer__image" alt="Youtube - @GetNinjas" src="/demo/gaiden-css/images/icons/youtube.png">
+            </a>
+          </li>
+        </ul>
       </div>
-    </section>
+    </nav>
+    <div class="container">
+      <ul class="footer__institutional col-small-12">
+        <li>
+          <a href="#"><img alt="GetNinjas" class="footer__image" src="/demo/gaiden-css/images/svg/getninjas.svg"></a>
+        </li>
+        <ul class="footer__institutional-list space-element-large">
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Imprensa" href="https://www.getninjas.com.br/imprensa">Imprensa</a></li>
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Ajuda" href="https://getninjas.desk.com/" target="_blank">Ajuda</a></li>
+          <li class="footer__item" role="none"><a role="menuitem" aria-label="Termos de uso" href="https://www.getninjas.com.br/termos-de-uso">Termos de uso</a></li>
+        </ul>
+      </ul>
+    </div>
+    <address role="conentinfo" class="footer__copyright">©2011-2018, GetNinjas Serviços de Internet. CNPJ 14.127.813/0001-51 - Av. Rebouças, 2472 - Pinheiros São Paulo - SP</address>
```

Comment:
@eduardojmatos `conentinfo`

Reply:
@iondrimba nice. Our home is with this error...